pelicanconf.py

```diff
@@ -38,13 +37,12 @@
 
 # Feed (RSS/Atom) settings
 FEED_DOMAIN = SITEURL
-FEED_RSS = FEED_RSS_URL = "blog/feed.rss"
-FEED_ATOM = FEED_ATOM_URL = "blog/atom.xml"
+FEED_ATOM = "feeds/atom.xml"
+FEED_RSS = "feeds/feed.rss"
```

Comment:
I want to double check on the base template too, I think the base template might have hardcoded the wrong URLs in at some point or something, as there are 2 extra feeds listed in the head.